### ISSUE TITLE
ACCESSIBILITY audit using WAVE.WEBAIM.ORG

### ISSUE THREAD

**Issue by gradyrobbins** [gradyrobbins/gradyrobbins.github.io#94]
Here's a [link](https://www.a11yproject.com/checklist/)

**Comment by gradyrobbins** [comment]
via a11yproject's [github](https://github.com/a11yproject) account:  **colorable**

- [ ] Color combination contrast tester

**Comment by gradyrobbins** [comment]
<img width="1427" alt="wave audit sept 2021" src="https://user-images.githubusercontent.com/38964004/135020141-61e2fe44-7c4a-4c1d-b47b-bb6c4b94f833.png">

**Comment by gradyrobbins** [comment]
Errors:  
**Linked image missing alternative text** 

**What It Means**
An image without alternative text results in an empty link.
**Why It Matters**
Images that are the only thing within a link must have descriptive alternative text. If an image is within a link that contains no text and that image does not provide alternative text, a screen reader has no content to present to the user regarding the function of the link.
**What To Do**
Add appropriate alternative text that presents the content of the image and/or the function of the link.

**Comment by gradyrobbins** [comment]
Reduced 14 errors => 3 errors

**Comment by gradyrobbins** [comment]
**6 X Very low contrast Errors**
**What It Means**
Very low contrast between text and background colors.
**Why It Matters**
Adequate contrast of text is necessary for all users, especially users with low vision.
**What To Do**
Increase the contrast between the foreground (text) color and the background color. Large text (larger than 18 point or 14 point bold) does not require as much contrast as smaller text.
**The Algorithm... in English**
Text is present that has a contrast ratio less than 4.5:1, or large text (larger than 18 point or 14 point bold) has a contrast ratio less than 3:1. WCAG requires that page elements have both foreground AND background colors defined (or inherited) that provide sufficient contrast. When text is presented over a background image, the text must have a background color defined (typically in CSS) that provides adequate text contrast when the background image is disabled or unavailable. WAVE does not identify contrast issues in text with CSS transparency, gradients, or filters.

**Comment by gradyrobbins** [comment]
### Congratulations! No errors were detected! Manual testing is still necessary to ensure compliance and optimal accessibility.

<img width="717" alt="Screen Shot 2021-10-03 at 6 00 44 PM" src="https://user-images.githubusercontent.com/38964004/135774394-2bb6dafd-afb0-4bb1-9b8e-181573d06960.png">

**Comment by gradyrobbins** [comment]
**How to improve your site's performance and prevent security vulnerabilities**

Add rel="noopener" or rel="noreferrer" to each link identified in your Lighthouse report. 

In general, when you use target="_blank", always add rel="noopener" or rel="noreferrer":


```
<a href="https://examplepetstore.com" target="_blank" rel="noopener">
  Example Pet Store
</a>
```
